resources/prometheus/prometheus-rules.yaml

```diff
@@ -639,3 +716,96 @@ spec:
+          annotations:
+            message: "Latency error budget exhaustion for central's GRPC API. Current exhaustion: {{ $value | humanizePercentage }}."
+          expr: |
+            (central:grpc_server_handling_seconds:rate10m:p90:sample_count_filter28d * central:grpc_server_handling_seconds:rate10m:p90:error_budget_exhaustion28d) >= 0.5
+          labels:
+            service: central
+            namespace: "{{ $labels.namespace }}"
+            rhacs_instance_id: "{{ $labels.namespace }}"
+            grpc_service: "{{ $labels.grpc_service }}"
+            grpc_method: "{{ $labels.grpc_method }}"
+            severity: warning
+        - alert: Central latency error budget exhaustion for HTTP API - 90%
+          annotations:
+            message: "Latency error budget exhaustion for central's HTTP API. Current exhaustion: {{ $value | humanizePercentage }}."
+          expr: |
+            (central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90:sample_count_filter28d * central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90:error_budget_exhaustion28d) >= 0.9
+          labels:
+            service: central
+            namespace: "{{ $labels.namespace }}"
+            rhacs_instance_id: "{{ $labels.namespace }}"
+            path: "{{ $labels.path }}"
+            severity: critical
+        - alert: Central latency error budget exhaustion for HTTP API - 70%
+          annotations:
+            message: "Latency error budget exhaustion for central's HTTP API. Current exhaustion: {{ $value | humanizePercentage }}."
+          expr: |
+            (central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90:sample_count_filter28d * central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90:error_budget_exhaustion28d) >= 0.7
+          labels:
+            service: central
+            namespace: "{{ $labels.namespace }}"
+            rhacs_instance_id: "{{ $labels.namespace }}"
+            path: "{{ $labels.path }}"
+            severity: warning
+        - alert: Central latency error budget exhaustion for HTTP API - 50%
+          annotations:
+            message: "Latency error budget exhaustion for central's HTTP API. Current exhaustion: {{ $value | humanizePercentage }}."
+          expr: |
+            (central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90:sample_count_filter28d * central:http_incoming_request_duration_histogram_seconds_bucket:rate10m:p90:error_budget_exhaustion28d) >= 0.5
+          labels:
+            service: central
+            namespace: "{{ $labels.namespace }}"
+            rhacs_instance_id: "{{ $labels.namespace }}"
+            path: "{{ $labels.path }}"
+            severity: warning
+        - alert: Central latency burn rate for GRPC API
+          annotations:
+            message: "Latency  burn rate for central's GRPC API. Current burn rate per hour: {{ $value | humanize }}."
+          expr: |
```

Comment:
Why did you chose `0.5` as the burn rate threshold? That seems very low to me. Note that by definition, a burn rate of 1 means that the full error budget will be consumed after 28 days.